src/main/java/cz/muni/fi/pa165/bluebat/entity/Customer.java

```diff
@@ -35,31 +37,37 @@
     @Getter
     @Setter
     @Column(nullable = false)
+    @Past
     private LocalDate birthday;
 
     @Getter
     @Setter
     @Column(nullable = false)
+    @NotBlank
     private String name;
 
     @Getter
     @Setter
     @Column(nullable = false)
+    @NotBlank
     private String email;
```

Comment:
Narazil jsem na to, že "@Email validates that the annotated property is a valid email address.", tak by to zde byl pravděpodobně lepší způsob validace.

Reply:
toho jsem si nějak nevšiml, dík